src/gui/addnewtorrentdialog.cpp

```diff
@@ -362,27 +356,6 @@ void AddNewTorrentDialog::showEvent(QShowEvent *event)
     raise();
 }
 
-void AddNewTorrentDialog::showAdvancedSettings(bool show)
-{
-    const int minimumW = minimumWidth();
-    setMinimumWidth(width()); // to remain the same width
-    if (show) {
-        m_ui->toolButtonAdvanced->setText(QString::fromUtf8(C_UP));
```

Comment:
I think the `C_UP` is unused now and can be deleted. Same with `C_DOWN`. This means that the `unicodestrings.h` include is probably unnecessary too.

Reply:
>I think the C_UP is unused now and can be deleted. Same with C_DOWN. 

OK, done.

>This means that the unicodestrings.h include is probably unnecessary too.

There are strings still in use, so no.

Reply:
I meant: remove the include from the `addnewtorrentdialog.cpp` file. It builds without it for me.

Reply:
>I meant: remove the include from the addnewtorrentdialog.cpp file. It builds without it for me.

Oh, it is done now.